.gitlab/pipelines/stages/deploy/stop_k8s.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+ps ax | awk '/[k]3s server/{ print $1 }' | xargs --max-line=1 --no-run-if-empty sudo kill
```

Comment:
You may have `pgrep`, actually. Both `pgrep` and `ps` are in the `procps` package.
```suggestion bash
pgrep --full 'k3s server' | xargs --no-run-if-empty sudo kill
```